frontend/www/js/omegaup/components/certificate/Validation.test.ts

```diff
@@ -0,0 +1,41 @@
+import { shallowMount } from '@vue/test-utils';
+
+import T from '../../lang';
+
+import certificate_Validation from './Validation.vue';
+
+describe('Validation.vue', () => {
+  it('Should handle an invalid certificate', async () => {
```

Comment:
nit:
```suggestion
  it('Should handle an invalid certificate', () => {
```
en esta prueba no requerimos que sea asíncrona porque no tiene que esperar ninguna acción